test/config/integration/server_xds.lds.typed_struct.yaml

```diff
@@ -0,0 +1,22 @@
+version_info: "0"
```

Comment:
This is a bit strange; we shouldn't need to have xDS resource objects embedded in typed struct, I don't think we plan on doing this on the wire.

Reply:
Where would be a more appropriate place to test this? I just wanted to make easy end to end test  .

Reply:
@htuch the tested TypeStruct here is in filter chain which we do typed struct, no?

Reply:
OK, let's go with this for now. I'd like to avoid as much as possible copy+paste of any of these files as it means we need to maintain two files in parallel, but if they have a bounded sunset (i.e. are deprecated in your other PR), then sure.